app/electron/preload.ts

```diff
@@ -57,4 +58,27 @@ contextBridge.exposeInMainWorld('desktopApi', {
   removeListener: (channel: string, func: (...args: unknown[]) => void) => {
     ipcRenderer.removeListener(channel, func);
   },
+
+  // MCP client APIs
```

Comment:
Can you please help me understand the security requirements of each of the APIs?

Which of these:
- should be off by default
- should be restricted to the ai-assistant plugin
- should require electron side confirmation